.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,4 +23,9 @@ Closes #
 
 #### Do we need any specific form for testing your changes? If so, please attach one.
 
-#### Does this change require updates to documentation? If so, please file an issue at https://github.com/opendatakit/docs/issues/new and include the link below.
\ No newline at end of file
+#### Does this change require updates to documentation? If so, please file an issue at [Here]( https://github.com/opendatakit/docs/issues/new) and include the link below.
+
+#### Before submitting this PR, please make sure you have tried:
+- [ ] Run the tests with `./gradlew pmd checkstyle lint findbugs` to make sure you didn't break anything.
```

Comment:
YES, that's definitely a good one. "to make sure you didn't break anything." sounds a little aggressive, how about "to make sure all checks still pass"?